share/eds/cache/accessor_cache.go

```diff
@@ -180,11 +159,11 @@ type accessorCloser struct {
 	io.Closer
 }
 
-func newCloser(abs *accessorWithBlockstore) (*accessorCloser, error) {
+func newCloser(abs *accessorWithBlockstore) *accessorCloser {
```

Comment:
[nit] another closer wants to be moved to utils

Reply:
This one is is noop closer, that will be replaced in following PR with ref counter, that will live in this file.

Reply:
I see, ok